src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewMapper.xml

```diff
@@ -91,8 +91,8 @@
 
     <select id="getSampleClinicalDataFromStudyViewFilter" resultType="org.cbioportal.model.ClinicalData">
         SELECT
-            sample_unique_id as sampleId,
-            patient_unique_id as patientId,
+            replaceOne(sample_unique_id, concat(cancer_study_identifier, '_'), '') as sampleId,
+            replaceOne(patient_unique_id, concat(cancer_study_identifier, '_'), '') as patientId,
```

Comment:
This should probably be refactored at some point as stated here
https://github.com/cBioPortal/cbioportal/blob/9a5f9e3b7d905890fdd5796118ba6a1257eb4cd3/src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewMapper.xml#L144-L147